lisa/tools/iperf3.py

```diff
@@ -131,6 +132,8 @@ def run_as_server_async(
         process = self.node.execute_async(
             f"{self.command} {cmd}", shell=True, sudo=True
         )
+        if 0 == len(self.node.tools[Pgrep].get_processes(self.command)):
```

Comment:
How about to call process.wait_result, if it happened. So there have more information get from results, like exitcode and stdout.

Reply:
if call process.wait_result here, it is not run as async mode?

Reply:
Call it in the `if` block, so it's only called when the process doesn't exist, it means it's failed already. That should be very fast.

Reply:
check process existing or not is not a good way, since the iperf3 may be launched multiple times. not sure check process is running or not is a good way or not, running testing now.

Reply:
just keep fix part for client, need figure out a way to fix issue when output is json format.